R/dockerfile.R

```diff
@@ -27,38 +19,130 @@
 #' dockerfile()
 #'
 #' @import futile.logger
-dockerfile <- function(from = sessionInfo(), to = paste0(getwd(), "/", "Dockerfile")) {
-  flog.debug("Creating a new Dockerfile from %s to %s", from, to)
+dockerfile <- function(from = utils::sessionInfo(),env = NULL, maintainer = NULL, r_version = paste(R.Version()$major, R.Version()$minor, sep="."), image = NULL, context = NA_character_) {
```

Comment:
- This would, again, not match the original intention of 'env' (see above), but we can discuss that
- I thought about this. Currently the program has the following behavior:

1. User specifies nothing --> image is determined by the R version argument, which by default the current version. R version matches the tag of rocker/r-ver
2. User specifies his own image (NULL, by default). Containerit will not try to find an image from the version, but takes anything the user gives

So, effectively, only one of these arguments can be evaluated in order to determine an image. Then, again, we could ask what should happen if a user supplies a sessionInfo-object from a different R instance than the current and wants to dockerize this.